stdlib/pathlib.pyi

```diff
@@ -38,7 +39,11 @@ class PurePath(PathLike[str]):
     def suffixes(self) -> list[str]: ...
     @property
     def stem(self) -> str: ...
-    def __new__(cls, *args: StrPath) -> Self: ...
+    if sys.version_info >= (3, 12):
+        def __new__(cls, *args: StrPath, **kwargs: Unused) -> Self: ...
```

Comment:
Technically `*args` are also unused in `__new__`, but I fear that using `Unused` here might break `*args: StrPath` in `__init__`. So, I went with the safest bet.